anycloud/cli/src/common/mod.rs

```diff
@@ -0,0 +1,99 @@
+use std::env;
+use std::fs::read;
+use std::process::Command;
+
+pub fn get_base_agz_b64() -> String {
+  base64::encode(include_bytes!("../../alan/anycloud.agz"))
```

Comment:
I wonder if it's possible to make this a `const` and have the compiler do it at build time? I'm not sure what restrictions are there, but it seems like it would be more efficient? @cdmistman ?

Reply:
unfortunately, it'd require <https://docs.rs/base64/0.13.0/base64/fn.encode.html> to be a `const fn`... we could have `build.rs` encode the `agz` file and output it to another file so we can do
```rust
const ANYCLOUD_AGZ: &'static str = include_str!("../../alan/anycloud.agz.b64");
```
Do we absolutely need to b64-encode, anyways?

Reply:
Yeah, we do, because it's passed over as a key-val pair in a JSON blob.

Reply:
ahhhhh right... yeah, the best we can do for now is to do the encoding with `build.rs`. I don't think there's anything blocking a `const fn` variant or a proc-macro of `base64::encode` though, so I wonder if we might be able to contribute it or at least open a feature request?

Reply:
It was a passing comment to make this slightly more efficient, but not really needed.

I think opening a feature request for `const fn` versions of `encode` (and maybe `decode`?) is probably a good idea, though?